src/analyses/base.ml

```diff
@@ -73,7 +73,6 @@ struct
 
   let name () = "base"
   let startstate v: store = { cpa = CPA.bot (); deps = Dep.bot (); weak = WeakUpdates.bot (); priv = Priv.startstate ()}
-  let otherstate v: store = { cpa = CPA.bot (); deps = Dep.bot (); weak = WeakUpdates.bot (); priv = Priv.startstate ()}
```

Comment:
The `otherstate` in `control.ml` is related to `otherfuns`, so that name is appropriate (until we get a better way to harness Linux kernel benchmarks).
The `otherstate` in analysis was a separate thing, which was replaced by `threadenter`.